_data/government.yml

```diff
@@ -11,6 +11,15 @@ websites:
       img: healthcaregov.png
       tfa: No
 
+    - name: RealMe
+      url: https://www.realme.govt.nz/
+      img: realme.png
+      tfa: Yes
+      sms: Yes
+      hardware: Yes
```

Comment:
What kind of hardware 2fa does it support? It seems like only sms and google authenticator.